domain/lib/src/model/sort/order_screen.dart

```diff
@@ -29,5 +29,6 @@
 //  3 and <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf> for
 //  the Additional Terms applicable to LinShare software.
 enum OrderScreen {
-  mySpace
+  mySpace,
+  shareSpace
```

Comment:
Use Enum Syntax please : `MY_SPACE, SHARED_SPACE`
You forgot the D aswell

Reply:
Well for Enum it's not recommanded we should refactor all enum then : https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names

Reply:
I think it's too cool. You should read it here [https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names](https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names)

Reply:
Yh lol just sent you the same link, I'll refactor my enum now, sorry but still add a D to sharedSpace pls aha

Reply:
Yes, I will correct it